src/Output.cs

```diff
@@ -2,124 +2,133 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+using Newtonsoft.Json;
+using System.Linq;
 
 namespace ACConfigBuilder
 {
     public class Output
     {
-        public void getobject(ACConfig AC, string path)
+        public dynamic writeOutput(List<string> AC, dynamic s)
```

Comment:
geb hier am besten kein dynamic rein. Dazu ist der Code unten zu unsafe. Mach es hier am besten so das du ein StreamWriter als object erwartest. dann kannst du auch das return dir sparen:

```suggestion
        public void writeOutput(List<string> AC, StreamWriter s)
```